tsdb/db.go

```diff
@@ -899,7 +903,10 @@ func (db *DB) run() {
 			case db.compactc <- struct{}{}:
 			default:
 			}
+		case <-scheduledCompact.C:
```

Comment:
This does nothing: Go `switch` cases do not fall through by default.

Reply:
We should be calling `compactOOOHead` under this actually (with the cmtx lock)